vue.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       }
     }
   },
-  baseUrl: process.env.NODE_ENV === 'production'? '/onnxjs-demo/': '/',
+  publicPath: process.env.NODE_ENV === 'production'? '/onnxjs-demo/': '/',
```

Comment:
I probably don't have any context about this - but do you know where this change comes from ?

Reply:
in latest Vue, `baseUrl` is deprecated and is replaced by `publicPath`.

Reply:
I see. This is because of vue update then.